padle/padle.py

```diff
@@ -210,7 +211,7 @@ async def start(self, ctx):
             return await send_cancellation_message(ctx,
                                                    "Looks like I can't DM you. Try checking your Privacy Settings.")
         if ctx.guild is not None:
-            await send_confirmation_message(ctx, "Check your DMs!")
+            await send_confirmation_message(ctx, f"{ctx.author.name}, check your DMs!")
```

Comment:
Does `send_confirmation_message` accept markdown? Does the author name need to be sanitized?

Reply:
As long as it's not unsafe, it's fine to me.  If someone wants their discord name to be __TheKing__ or whatever they have to deal with it.  I've never put a clause for that in my code before.